bin/paragon-scripts.js

```diff
@@ -106,6 +106,16 @@ const COMMANDS = {
         description: 'Enable verbose logging.',
         defaultValue: false,
       },
+      {
+        name: '--base-theme',
+        description: 'Specify the base theme to use in the token build. For example, to build the "high-contrast" theme on top of the ligth theme use "--theme high-contrast --base-theme light".',
+        defaultValue: 'Same as theme',
+      },
+      {
+        name: '--all-themes',
```

Comment:
I like your idea of ​​adding the `--all-themes` option. This is much better for building all themes, instead of manually entering the name of each theme 💯